client/html/themes/aimeos.js

```diff
@@ -386,7 +386,13 @@ AimeosBasketMini = {
 	 */
 	update: function() {
 
-		$.ajax($(".aimeos.basket-mini[data-jsonurl]").data("jsonurl"), {
+		var jsonurl = $(".aimeos.basket-mini[data-jsonurl]").data("jsonurl");
+
+		if( typeof jsonurl === 'undefined' ) {
```

Comment:
Wouldn't it be better to check for an empty string?
```
if(jsonurl == '') {
  return;
}
```

Reply:
No, because the type is `undefined`, actually, and in JS `undefined != ''`. But I will add the empty string case, if a parameter is not set etc.